docker/mainnet/connector/connector.mainnet.conf

```diff
@@ -0,0 +1,54 @@
+[Application Options]
+datadir=/root/.connector
+debuglevel=debug
+network=mainnet
+# RPC address to bind
+rpchost=0.0.0.0
+rpcport=9002
+# exchange address
+enginehost=mainnet.exchange.bitlum.io
+engineport=8080
+
+[Bitcoin]
+bitcoin.minconfirmations=1
+bitcoin.syncdelay=5
+bitcoin.host=bitcoin.mainnet
+bitcoin.port=8332
+bitcoin.feeperunit=87
+
+[Bitcoincash]
+bitcoincash.minconfirmations=1
+bitcoincash.syncdelay=5
+bitcoincash.host=bitcoin-cash.mainnet
+bitcoincash.port=9332
+bitcoincash.feeperunit=350
+
+[Dash]
+dash.minconfirmations=1
+dash.syncdelay=5
+dash.host=dash.mainnet
+dash.port=10332
+dash.feeperunit=350
```

Comment:
We need to change this accordingly with real fee per unit and later make this dynamic

Reply:
What is our real `fee per unit`?